src/news/release-1.3.2.md

```diff
@@ -0,0 +1,41 @@
+---
+title: PolyMC Update 1.3.2, now available
+description: Another load of fixes
+date: 2022-06-12
+tags:
+  - release
+---
+
+Yet again we have some fixes for you!
+
+## Changelog
+
+### Added
+- Add support for Quilt modpacks on Technic by @kthchew in #685
+- [Linux] Added support for Nix 2.7 by @TheOPtimal in #286
+- [Windows] Add version information to setup executable by @kthchew in #679
+
+### Changed
+- Change build flags by @Scrumplex in #764
+- Do not enforce fractional scaling by @DioEgizio in #718
+- Sort icons in icon-selector alphabetically by @MrMelon54 in #732
+- [Linux] Allow adding more JDKs to Nix package by @babbaj in #734
+
+### Fixed
+- Allow importing .mrpack files on non-latin languages by @flowln in #780
```

Comment:
```suggestion
- Allow importing .mrpack files on non-latin languages by @flowln and @Scrumplex in #780
```

you deserve more credit for that than i tbh :)

Reply:
I already forgot that I did some work there as well